knexfile.js

```diff
@@ -0,0 +1,56 @@
+const localPg = {
+  host: 'localhost',
+  database: 'tracker',
+  user: 'null',
+  password: 'null'
+}
+const productDbConnection = process.env.DATABASE_URL || localPg;
+
+module.exports = {
+  development: {
+    client: 'sqlite3',
+    connection: {
+      filename: './src/data/tracker.db3',
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: './src/data/migrations',
+    },
+    seeds: {
+      directory: './src/data/seeds',
+    },
+    pool: {
+      afterCreate: (conn, done) => {
+        conn.run('PRAGMA foreign_keys = ON', done);
+      },
+    }
+  },
+  testing: {
+    client: 'sqlite3',
+    connection: {
+      filename: './src/data/test.db3',
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: './src/data/migrations',
+    },
+    seeds: {
+      directory: './src/data/seeds',
+    },
+    pool: {
+      afterCreate: (conn, done) => {
+        conn.run('PRAGMA foreign_keys = ON', done);
+      },
+    }
+  },
+  production: {
+    client: 'pg',
```

Comment:
Maybe it's because you're starting with postgres instead of converting all of your routes after using sqlite, but I'm really impressed w/ the postgres implementation on day 1.